content/command-line/label-drives.md

```diff
@@ -17,8 +17,12 @@ directpv label drives key=value|key- [flags]
 ```
 
 Use only one or the other of the options:
-- Use `key=value` to add a label `key` with the value of `value` to the drive(s).
-- Use `key-` to remove the label `key` from the drive(s).
+- Use `key=value` to add a custom label `key` with the value of `value` to the drive(s).
```

Comment:
Related: while you are in here, there's a couple other locations of "key=value" that could be `key=value`

Reply:
Done!